packages/js/src/browser.ts

```diff
@@ -40,19 +60,20 @@ class Honeybadger extends Client {
     }
   ]
 
-  constructor (opts: Partial<Types.BrowserConfig> = {}) {
-    const global = globalThisOrWindow()
-    let projectRoot = ''
-
-    // Cloudflare workers do not have access to location API.
-    if (global.location != null) {
-      projectRoot = global.location.protocol + '//' + global.location.host
+  protected __afterNotifyHandlers: Types.AfterNotifyHandler[] = [
```

Comment:
`afterNotify` handler to store last reported notice id, needed for the feedback form.